multiqc/1.12/multiqc_1.12.cwl

```diff
@@ -0,0 +1,84 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  sbg: 'https://www.sevenbridges.com/'
```

Comment:
We need to add additional namespace for the tags we used below to give author information

Reply:
done